src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+    <title>Document</title>
+</head>
+
+<body>
+
+    <div class="listContainer">
+        <div class="listblock">
+            <div class="Todo-section">
+                <h1>Todo List</h1>
+                <i class="fas fa-sync"></i>
+            </div>
+            <input type="text" class="textfield" placeholder="Add to your list...." id="inputTask">
```

Comment:
- according to the sample that is provided in the video you miss the enter icon to click on that icon to add new item list kindly add submit button.